app/views/usage/filter.js.erb

```diff
@@ -1,5 +1,9 @@
 // Using straight (non-jquery) JS here because the js.erb does not have access to JQuery!
 // TODO: We should fix this and load JS and other assets correctly when we move to Rails 5
 //       it would be much cleaner to just rerender the html.erb
-document.getElementById('filter_ranged').innerHTML = '<%= escape_javascript(render partial: "filtered_value", locals: { label: "New #{@topic}", value: @ranged.to_i }) %>';
-document.getElementById('filter_total').innerHTML = '<%= escape_javascript(render partial: "filtered_value", locals: { label: "Total #{@topic}", value: @total.to_i }) %>';
\ No newline at end of file
+document.getElementById('filter_ranged').innerHTML = '<%= escape_javascript(render partial: "filtered_value", 
+                                                    locals: { label: _("New %{topic}") % {topic: @topic }, 
+                                                    value: @ranged.to_i }) %>';
+document.getElementById('filter_total').innerHTML = '<%= escape_javascript(render partial: "filtered_value", 
+                                                    locals: { label: _("Total %{topic}") % {topic: @topic }, 
+                                                    value: @total.to_i }) %>';
```

Comment:
I too get the urge to split long lines of code this way. However, most of ruby's syntax calls for longer lines of code. For now we should leave them in a single line, compare to the rest of the code base, and set rubocop rules to enforce them.

The newer version of roadmap does have a functioning rubocop workflow so we can address  these issues then.